Assets/Scripts/Localization/LocalizationTable.cs

```diff
@@ -120,6 +174,82 @@ public static string[] GetLanguages()
             return localizationTable.Keys.ToArray();
         }
 
+        public static void LoadConfigFile(string pathToConfigFile)
+        {
+            localizationConfigurations = new Dictionary<string, LanguageData>();
+
+            if (File.Exists(pathToConfigFile) == false)
+            {
+                Debug.ULogErrorChannel("LocalizationTable", "No config file found at: " + pathToConfigFile);
```

Comment:
Should this also be a Warning, since it is an expected result at certain times, and is remedied by creating one if it isn't found? Assuming I'm understanding the process of what's going on anyways.